clang/test/dpct/codepin/report/data_mismatch/CodePin_Report_ref.csv

```diff
@@ -2,4 +2,4 @@
 // CHECK: Totally APIs count, 2
 // CHECK: Consistently APIs count, 1
 // CHECK: CUDA Meta Data ID, SYCL Meta Data ID, Type, Detail
-// CHECK: serial123,serial123,Data value,The location of failed ID "var_2"->"Data"->"member_2"->"Data" and [ERROR: DATA VALUE MISMATCH] the CUDA value "bb123" differs from the SYCL value "ccc".
+// CHECK: serial123:prolog:0,serial123:prolog:0,Data value,[WARNING: METADATA MISMATCH] The pair of prolog data serial123:prolog:0 are mismatched, and the corresponding pair of epilog data matches. This mismatch may be caused by the initialized memory or argument used in the API serial123:prolog:0.
```

Comment:
serial123:prolog:0 is not API name
```

Reply:
Ok, I will submit another PR to follow this suggestion.